src/main/scala/jp/ne/opt/chronoscala/NamespacedImplicits.scala

```diff
@@ -0,0 +1,13 @@
+package jp.ne.opt.chronoscala
+
+import jp.ne.opt.chronoscala.Tag.CS
+
+import scala.language.implicitConversions
+
+trait NamespacedImplicits extends Implicits with NamespacedIntImplicits
+
+trait NamespacedIntImplicits extends IntImplicits {
+  override def richInt(n: Int): RichInt = new RichInt(n)
```

Comment:
That's ok.

Other things to consider is that should we create name spaced variants for other implicit conversions for `Duration`, `ZonedDateTime` etc. These are rarely conflicted, IMO.